### ISSUE TITLE
OVERLAP : Unclickable column NN : Hidden by the dropdown on its left

### ISSUE THREAD

**Issue by Raphael-MOUQUIN** [sqlitebrowser/sqlitebrowser#2763]
<img width="535" alt="scrshot_SQLiteStudio3 3 3_HIDDEN_COLUMN_NN" src="https://user-images.githubusercontent.com/86474028/124496902-5da9d980-ddba-11eb-9021-3befd6bd89da.PNG">

### Details
just clicked "New Database" then this "Edit table definition" window appeared but unable to click in the NN column due to the overlap of the dropdown on its left

### Steps to reproduce
Issue is always present  . 

### Operating system
Win7

#### What is your DB4S version?

3.12.1

**Comment by Raphael-MOUQUIN** [comment]
FYC:  
Custom GUI Font size "boosted" to 150% , wide-range, impact ALL GUI including Qt v.5.12.8 or is it something else_?

<img width="294" alt="PPP custom 144dpi ppi, Font Segoe 9 pts" src="https://user-images.githubusercontent.com/86474028/124628246-914f3700-de80-11eb-9163-400410b3e041.PNG">

**Comment by Raphael-MOUQUIN** [comment]
and it might be useful to spot the reason of that GUI' perk : 

especially regarding the **Font set & sizes** uded : 

<img width="489" alt="scrshot_DB Browser for SQLiteStudio-3 12 1_PREFERENCES_TAB_GENERAL" src="https://user-images.githubusercontent.com/86474028/124628703-0589da80-de81-11eb-838f-848e05bb41ad.PNG">


and

<img width="489" alt="scrshot_DB Browser for SQLiteStudio-3 12 1_PREFERENCES_TAB_DATABASE" src="https://user-images.githubusercontent.com/86474028/124628733-0fabd900-de81-11eb-8850-78b951285d74.PNG">

**Comment by chrisjlocke** [comment]
If you increase the size of the 'type' column, does it allow the 'NN' column to be shown fully?

**Comment by chrisjlocke** [comment]
@Raphael-MOUQUIN - Have you tried increasing the size of the 'type' column?

**Comment by mgrojo** [comment]
Same Qt bug as in  #2490.

**Comment by Raphael-MOUQUIN** [comment]
> If you increase the size of the 'type' column, does it allow the 'NN' column to be shown fully?

@chrisjlocke  i'm installing it again to try what you asked. Hang on

**Comment by Raphael-MOUQUIN** [comment]
@chrisjlocke  Right, you were right: Resizing/Expanding the colum 'type' **un**-overlap the drop-down combo so we can see the checkbox NN . Not Null 
<img width="304" alt="DB Browser for SQLite TABLE COLUMNS RESIZE" src="https://user-images.githubusercontent.com/86474028/126306023-ab64e89a-a431-4e58-97c7-f504d1b5e18c.PNG">

Thanks, you can close this issue

Best Regards
Raphael

**Comment by chrisjlocke** [comment]
I'll re-open this, as someone cleverer than me might be able to forcibly set the size of the type column to a slightly wider value.  This may or may not affect other people...

**Comment by mgrojo** [comment]
I already made that in 532457edb7abe88b0c4af9eeb4b48a05a4faaf06

Nevertheless, I consider that a workaround. It will still happen when some field is more than 150 px wide. The proper solution for both issues is updating to a newer Qt version.
